index.html

```diff
@@ -465,8 +468,10 @@ <h2>The <code><dfn>IdleDeadline</dfn></code> interface</h2>
         <a>IdleDeadline</a> object.</li>
         <li>Let <var>timeRemaining</var> be <var>deadline</var> -
         <var>now</var>.</li>
-        <li>If the user agent now has a time-critical task pending,
-        set <var>timeRemaining</var> to 0.</li>
+        <li>Call the <a>is high-priority work pending algorithm</a>, passing
+	the <a>event loop</a> associated with <var>window</var> as
+	<var>event_loop</var>. If it returns <code>true</code> set 
```

Comment:
We don't typically use underscored names for variable names in spec algorithms.

Reply:
Done